build/debs/BUILD

```diff
@@ -167,6 +174,12 @@ The Container Networking Interface tools for provisioning container networks.
     version_file = "//build:cni_package_version",
 )
 
+k8s_deb(
+    name = "cri_tools",
+    description = """Container Runtime Interface tools (crictl)""",
+    version_file = "//build:cri_tools_package_version",
```

Comment:
it isn't work using `version_file` if it never changes. just use the `version` attribute instead (which takes a string, rather than a file).